index.html

```diff
@@ -0,0 +1,671 @@
+                        </p>
+                    </div>
+                </div>
+                <!-- item-3 -->
+                <div class="funding-item">
+                    <div class="one">
+                        <img class="funding-image" src="images/klook.jpeg" alt="klook">
+                        <h3 class="pad-right">Klook</h3>
+                        <p class="pad-right"><strong>$200M</strong>
+                            <br>
+                            <span class="sub-head">SERIES A</span>
+                        </p>
+                    </div>
+
+                    <div class="three">
+                        <p><i class="fa fa-tag" aria-hidden="true"></i> Internet</p>
+                        <p><i class="fa fa-briefcase" aria-hidden="true"></i> Boyu Capital,+4</p>
+                    </div>
+                    <hr />
+                    <div class="four">
+                        <p>SoftBank-backed travel platform Klook raises $200M amid COVID-19
+                            <span class="grey-txt">on TechCrunch</span>
+                        </p>
+                    </div>
+                </div>
+                <!-- item-4 (visible when expand) -->
+                <div id="item-4" class="funding-item">
+                    <div class="one">
+                        <img class="funding-image" src="images/wolt.jpeg" alt="wolt">
+                        <h3 class="pad-right">Wolt</h3>
+                        <p class="pad-right"><strong>$530M</strong>
+                            <br>
+                            <span class="sub-head">VENTURE CAPITAL</span>
+                        </p>
+                    </div>
+
+                    <div class="three">
+                        <p><i class="fa fa-tag" aria-hidden="true"></i> Mobile +5</p>
+                        <p><i class="fa fa-briefcase" aria-hidden="true"></i> Dstv global, +9</p>
+                    </div>
+                    <hr />
+                    <div class="four">
+                        <p>Food delivery startup Wolt raises $530M after tripling revenues
+                            <span class="grey-txt">on SiliconANGLE</span>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="categories">
+        <div class="cat-item">
+            <div class="cat-heading">
+                <h1><a href="#">Hard Fork</a></h1>
+            </div>
+            <!-- <div class="cat-gradient"> -->
+            <div class="cat-img-box">
+              <img src="images/categories/hardfork.png" alt="game stop">
+            </div>
+            <div class="cat-img-txt">
+              <h4>MOST POPULAR</h4>
+              <P><a href="#" style="font-weight: bold; font-size: 20px; color: white">GameStop stocks hit record high — thanks to Redditors</a></P>
+              <p><a class="cat-grey-txt"href="#">Ivan Mehta</a><span class="grey-txt">. 1 day ago</span></p>
```

Comment:
- [ ] Use lowercase element names: you can see specific description [here](https://www.w3schools.com/html/html5_syntax.asp)
- [ ] Using inline styles is one of the [common mistakes](https://speckyboy.com/good-bad-css-practices). Styles should be defined as separated from html.